admin_manual/installation/system_requirements.rst

```diff
@@ -52,6 +52,7 @@ Server
 
 .. note::
 
+   - Ubuntu 18.04 ships with PHP 7.2. For production use, install PHP 7.1
```

Comment:
The table in this version of the file says that PHP 7.2 is supported.
So this line is no longer needed. People on Ubuntu 18.04 can use the default PHP 7.2

Reply:
Yep, I'm removing that in a new PR.